### PR TITLE
Basic portfolio website : Use media queries to make sure that website displays correctly on desktop screens. (Desktop: 768px or wider).

### PR DESCRIPTION
**Pull request by EmekaEmmanuel**
Portfolio Setup is a simple portfolio website with these sections contained in this project;
- Toolbar
- Headline section (right after the header).
- Works
- About myself
- Contact me
The layout was done using css flexbox and css grid, button interactions implemented.
Use media queries to make sure that website displays correctly on desktop screens. (Desktop: 768px or wider).

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -12,61 +12,87 @@
...
+                </ul>
 
-        </div>
+            </div>
+
+        </article>
 
     </section>
 
     <section class="works_section">
```
Comment (amejid) [inline index.html:79]: - [ ] Good job applying media queries to make the website responsive. While doing so we should also make sure that we maintain the mobile layout. 🥸️
Kindly update the gap between your projects and the margin on the sides as per the Figma template.

| Your design                                                             | Figma design                                         |
| --------------------------------------------------------------------------------| --------------------------------------------------------- |
|||

File: index.html [hunk 2]
```diff
@@ -12,61 +12,87 @@
...
+                    <li><a href="#"><img src="./img/mee.jpg" alt="mee_image"></a></li>
+                    <li><a href="#"><img src="./img//twitter.jpg" alt="twitter_image"></a></li>
 
-            </ul>
+                </ul>
 
-        </div>
+            </div>
+
+        </article>
```
Comment (amejid) [inline index.html]: - [ ] One of the objectives of this project is to understand how to parse Figma templates precisely. 🤝️
Kindly update the gap between the paragraph and the social icons to match the Figma design.

- Your Design
Screenshot from 2022-12-08 20-21-18

- Figma Design
Screenshot from 2022-12-08 20-20-35

File: index.html [hunk 3]
```diff
@@ -94,26 +120,55 @@ <h3><a class="textdecoration font1" href="#">Tonic</a></h3>
...
+                    <li class="background1"><a class="textdecoration font4" href="#">html</a></li>
+                    <li class="background1"><a class="textdecoration font4" href="#">Ruby on rails</a></li>
+                    <li class="background1"><a class="textdecoration font4" href="#">css</a></li>
+                    <li class="background1"><a class="textdecoration font4" href="#">javaScript</a></li>
+
+                </ul>
+
+                <button type="submit"><a class="font5 textdecoration" href="#">See Project</a></button>
+
+            </div>
```
Comment (amejid) [inline index.html:150]: - [ ] Good job alternating the image and project description. I think there's a little more to do to match the Figma design.
Kindly check the whitespace to the left and right of the component. 😎️

- Your Design
Screenshot from 2022-12-08 20-26-00

- Figma Design
Screenshot from 2022-12-08 20-25-04

File: index.html [hunk 4]
```diff
@@ -208,34 +335,77 @@ <h3><a class="textdecoration font1" href="#">Multi-Post Stories</a></h3>
...
+                <a href="#"><img src="./img/jumbo2b.png" alt="jumbo4_img"></a>
+
+            </figure>
+
+        </article>
 
     </section>
 
-    <section class="about_self_section">
+    <section class="about_self_section bg_color7">
```
Comment (amejid) [inline index.html:378]: Nice job so far on the about section. I think some small details are missing when compared to the Figma template. ✍️
- [ ] Kindly use `40%` width on the left section and `60%` on the right one to achieve the Design.
- [ ] Kindly increase the width of the cards that hold HTML, CSS, and Javascript icons and make the icons align to the left inside the card. Please update the text `color` below the icon images.
- [ ] I think a fixed with is being applied to the paragraph in the left section. Kindly update the `font-size` and remove the fixed width so the paragraph can span the whole parent element.

- Your Design
Screenshot from 2022-12-08 20-30-52

- Figma Design
Screenshot from 2022-12-08 20-30-13

File: index.html [hunk 5]
```diff
@@ -286,7 +461,57 @@ <h5>LET'S CONNECT</h5>
...
             </ul>
 
-        </div>
+        </article>
+
+    </section>
+
+    <section class="bg_color7">
+
+        <article class="contact_section">
```
Comment (amejid) [inline index.html:469]: You are almost there with the design. There are some small details remaining to achieve perfection. 🤓️
- [ ] Kindly check if the correct background image for this section is being used.
- [ ] Kindly make sure the paragraph and the `form` element are positioned in the center of the screen. You can give these elements a `max-width` property to achieve this alignment.

- Your Design
Screenshot from 2022-12-08 20-33-11

- Figma Design
Screenshot from 2022-12-08 20-32-35

**Review by amejid — CHANGES_REQUESTED** [review]
Hi @EmekaEmmanuel, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
- Correct use of Github flow :heavy_check_mark: 
- Good commit messages :heavy_check_mark: 
- Linter checks passing :heavy_check_mark: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by DuaneDave — APPROVED** [review]
Hi @EmekaEmmanuel 👋 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
✅ Solid design 👍
✅ Quick implementation of requested change 👍 
✅ Meaningful commit history 👍 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code. Some of them were missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**